test/unit/jasmine-boot.js

```diff
@@ -60,7 +60,6 @@ function initializePDFJS(callback) {
     'pdfjs-test/unit/dom_utils_spec',
     'pdfjs-test/unit/encodings_spec',
     'pdfjs-test/unit/evaluator_spec',
-    'pdfjs-test/unit/fonts_spec',
```

Comment:
I forgot to mention it earlier, but you'll also need to remove the following line: https://github.com/mozilla/pdf.js/blob/436d2efa8a89edef3a05f70592e3a70fd0c7d11e/test/unit/clitests.json#L21